python/meterpreter/ext_server_stdapi.py

```diff
@@ -1216,6 +1217,85 @@ def stdapi_sys_config_getuid(request, response):
     response += tlv_pack(TLV_TYPE_USER_NAME, username)
     return ERROR_SUCCESS, response
 
+@register_function_if(has_windll)
+def stdapi_sys_config_getprivs(request, response):
+    TOKEN_QUERY = 0x0008
```

Comment:
It's probably time to move this up to the Windows constants section around L735 so it doesn't need to be defined here and in `get_token_user`.

Reply:
Moved the constants around with https://github.com/rapid7/metasploit-payloads/pull/607/commits/ca738c412c1b0e76306e8d4e1015c5a571509bf8